Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openjdk:15
+
+MAINTAINER REM Java 21 1 <rafael.armbrust@gmail.com>
+
+ADD backend/target/github-bingo-master.jar app.jar
```

Comment:
wrong path name for .jar file i guess :)

Reply:
done